cmd/aws-vpc-cni/main.go

```diff
@@ -99,6 +100,8 @@ const (
 	envEnIPv6                = "ENABLE_IPv6"
 	envEnIPv6Egress          = "ENABLE_V6_EGRESS"
 	envRandomizeSNAT         = "AWS_VPC_K8S_CNI_RANDOMIZESNAT"
+	envEnableNftables        = "ENABLE_NFTABLES"
+	envIPCoolingPeriod       = "IP_COOLING_PERIOD"
```

Comment:
This should be `IP_COOLDOWN_PERIOD`, as that's what the README calls it. Let's also use `envIPCooldownPeriod`